UndertaleModTool/MainWindow.xaml.cs

```diff
@@ -1380,21 +1556,32 @@ private void DeleteItem(UndertaleObject obj)
             if (MessageBox.Show("Delete " + obj.ToString() + "?" + (!isLast ? "\n\nNote that the code often references objects by ID, so this operation is likely to break stuff because other items will shift up!" : ""), "Confirmation", MessageBoxButton.YesNo, isLast ? MessageBoxImage.Question : MessageBoxImage.Warning) == MessageBoxResult.Yes)
             {
                 list.Remove(obj);
-
-                if (obj is UndertaleCode codeObj)
+                for (int i = 0; i < Tabs.Count; i++)
```

Comment:
Add `if (obj is UndertaleCode codeObj)` and its content right after `list.Remove(obj);`.